### PR TITLE
Virtual env inst

### PR DESCRIPTION
**Pull request by puneith**
@jonparrott Please review.

### CODE REVIEW

File: speech/api/README.md [hunk 1]
```diff
@@ -38,6 +38,21 @@ See the
 [Cloud Platform Auth Guide](https://cloud.google.com/docs/authentication#developer_workflow)
 for more information.
 
+### Install pip
```
Comment (jonparrott) [inline speech/api/README.md]: I would prefer you to consolidate these steps like the [app engine flex readme](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/appengine/flexible#run-locally). Feel free to link to virtualenv and pip, but there's no need to give explicit instructions on installation.

File: speech/api/README.md [hunk 2]
```diff
@@ -38,6 +38,22 @@ See the
...
+
+* Clone this repo
+    ```
+    git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
+    cd python-docs-samples/speech/api
+    ```
+
+* Create virtualenv
+    ```
+    virtualenv speechsamples
```
Comment (jonparrott) [inline speech/api/README.md]: We usually just use 'env' as the virtualenv name.

**Comment by jonparrott** [comment]
Lgtm after final nit.

**Comment by puneith** [comment]
Ok fixed to env.

**Pull request merged**